src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -354,22 +358,27 @@ JournalTrimmerImpl::config_t::get_default(
   assert(roll_size);
   std::size_t target_dirty_bytes = 0;
   std::size_t target_alloc_bytes = 0;
+  std::size_t min_journal_bytes = 0;
   std::size_t max_journal_bytes = 0;
   if (type == backend_type_t::SEGMENTED) {
-    target_dirty_bytes = 12 * roll_size;
+    min_journal_bytes = 12 * roll_size;
     target_alloc_bytes = 2 * roll_size;
+    target_dirty_bytes = 14 * roll_size;
     max_journal_bytes = 16 * roll_size;
   } else {
     assert(type == backend_type_t::RANDOM_BLOCK);
-    target_dirty_bytes = roll_size / 4;
+    min_journal_bytes = roll_size / 4;
     target_alloc_bytes = roll_size / 4;
+    target_dirty_bytes = roll_size / 3;
     max_journal_bytes = roll_size / 2;
   }
   return config_t{
     target_dirty_bytes,
     target_alloc_bytes,
+    min_journal_bytes,
     max_journal_bytes,
-    1<<17,// rewrite_dirty_bytes_per_cycle
+    1<<26,// rewrite_dirty_bytes_per_cycle
+    1<<17,// rewrite_dirty_bytes_per_trans
     1<<24 // rewrite_backref_bytes_per_cycle
```

Comment:
Can we take the chance to comment these with human readable sizes? (MiB/KiB/etc.)